### DISCUSSION TITLE
What do you think about Codecov ?

### DISCUSSION THREAD

**Discussion by sivchari** [pipe-cd/pipecd#4539] [Q&A]
I've confirmed [this discussion](https://github.com/pipe-cd/pipecd/discussions/4470#discussioncomment-6464896), but I think Codecov affects a not good experience than good it.  Of course, I know Codecov is good product, but, in fact, some pull requests are failed by Codecov's reporting, although PipeCD maintainer comments that it's ok to go, now. wdyt ?

**Comment by khanhtc1202** [comment]
Well, the point of using that codecov is because some awesome list we're willing to join requires public test coverage. So we still should adopt that. Even we say okay to merge the PR, which reduces the test coverage, it doesn't mean we should turn it off. I will create tasks to increase PipeCD test coverage and can say for sure that we want to increase that number (and make it green)
